Resources/Prototypes/_Shitmed/Entities/Debugging/debug_sweps.yml

```
@@ -0,0 +1,82 @@
+        Heat: 20000
+    clickPartDamageMultiplier: 10
+  - type: Item
+    size: Tiny
+    sprite: Objects/Weapons/Melee/debug.rsi
+
+- type: entity
+  name: bang burner 200dmg
+  parent: MeleeDebugBurner
+  id: MeleeDebugBurner200
+  components:
+  - type: Tag
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)